prov/gni/src/gnix_nameserver.c

```diff
@@ -174,7 +174,8 @@ int gnix_resolve_name(IN const char *node, IN const char *service,
 	};
 
 	if (!resolved_addr) {
-		GNIX_LOG_ERROR("Resolved_addr must be a valid pointer.\n");
+		GNIX_ERR(FI_LOG_FABRIC,
```

Comment:
I wasn't sure what something like gnix_nameserver.c belonged under, is that FI_LOG_FABRIC as the subsystem? That was my best guess...